chore: change text in contributing file (#3524)
DevExpress/devextreme-reactive: CONTRIBUTING.md

# Contribution Guidelines

> ## Bugs and Feature Requests

Reviewer: ```suggestion
## Bug and Feature Requests
```

> If you found a bug or want to request a feature, please submit an issue to our repository. You should check the list of active issues first to avoid duplicates.

Reviewer: ```suggestion
Submit an issue to request a feature or report a bug. Check the list of active issues first to avoid duplicates.
```

## Help and Support

> If you are using DevExtreme Reactive under the DevExtreme Non-Commercial license, please refer to Stack Overflow for support. If you are the owner of an active commercial license, create a ticket in our [Support Center](https://www.devexpress.com/ask).

Reviewer: ```suggestion
Create a ticket in our [Support Center](https://www.devexpress.com/ask) if you own an active commercial license for DevExtreme Reactive. If you have a non-commercial license, refer to Stack Overflow for support.
```
